src/main/resources/reports/inputReport.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
+    b,banana,20

```

Comment:
```suggestion
b,banana,20
```
Let’s remove redundant whitespaces. Also, let’s  add an output  file to your solution